documentation/debugging.md

```diff
@@ -25,3 +25,7 @@ $ DEBUG=jsonApi:handler:find npm test
 ```
 $ DEBUG=jsonApi:handler:* npm test
 ```
+
+### Integration with application logging
+
+If you wish to integrate `jsonapi-server` debug output with your application's logging solution, you can override the functions that are invoked for generating the output of each of the supported namespaces by invoking the package's `debugging.__setLoggers(loggerFactory)` function where the `loggerFactory` parameter is a function that expects a namespace string parameter and returns a logger function for the namespace.
```

Comment:
Could you put an example beneath this? It's not entirely obvious by reading this paragraph and the code is both elegant and not immediately obvious.

In other words - people will ask us a lot of questions about how this works, so lets just add an example we can refer them back to.